### PR TITLE
Ads: Update flag and bower dependancies

### PR DESCRIPTION
**Pull request by imranolas**

### CODE REVIEW

File: client/main.js [hunk 1]
```diff
@@ -23,7 +23,7 @@ setup.bootstrap(({flags}) => {
 		selector: '.js-video'
 	});
 
-	if(flags.flags.frontPageAds) {
+	if(flags.get('frontPageAds')) {
```
Comment (commuterjoy) [inline client/main.js]: Are frontpage ads different to any other ads? If not, best us the existing 'ads' flag imho.

Comment (matthew-andrews) [inline client/main.js]: presumably they're doing this in order to deploy the code without enabling this feature?

Comment (imranolas) [inline client/main.js]: Yes. Just wanted to QA in relative safety before releasing to live.

**Comment by charypar** [comment]
The build failures are weird, need to investigate in the morning, if it continues...

**Pull request merged**